modules/library/main/src/main/java/org/geotools/util/CommonsConverterFactory.java

```diff
@@ -220,8 +220,8 @@
     }
     
     static class DateConverter implements Converter {
-        private static SimpleDateFormat format1 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.S a" );
-        private static SimpleDateFormat format2 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ssa" );
+        private SimpleDateFormat format1 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.S a" );
```

Comment:
Just checked, these converters are created once and saved into a static hash map, from which they are grabbed and used. So... moving the converters from static field to field does not hlpe, the DateConverter instance itself is static. Believe we need to have them created inside the conversion method instead.